src/simtools/production_configuration/derive_corsika_limits.py

```diff
@@ -217,12 +217,23 @@ def compute_lower_energy_limit(self, loss_fraction):
         astropy.units.Quantity
             Lower energy limit.
         """
-        return (
+        energy_min = (
             self._compute_limits(
                 self.histograms.get("energy"), self.energy_bins, loss_fraction, limit_type="lower"
             )
             * u.TeV
         )
+        return self._is_close(
+            energy_min,
+            self.file_info["energy_min"].to("TeV") if "energy_min" in self.file_info else None,
+            "Lower energy limit is equal to the minimum energy of",
+        )
+
+    def _is_close(self, value, reference, warning_text):
+        """Check if the value is close to the reference value and log a warning if so."""
+        if reference is not None and np.isclose(value.value, reference.value):
```

Comment:
Do you want to specify a tolerance here or we go with the default of 1e-05?

Reply:
I think for all application here (view cone, energy range, scatter area) this value is good - we essentially have to catch only the case when we run against the limit.